### ISSUE TITLE
Improve toast notifications design

### ISSUE THREAD

**Issue by samuveth** [snapshot-labs/snapshot#799]

**Comment by bonustrack** [comment]
Would "View on Explorer" be "View on IPFS" or what information you would like to see @samuveth?

**Comment by samuveth** [comment]
Yes, this can depend on what the action is. I will try to spec this a bit once I finished some of the other PRs

**Comment by midgerate** [comment]
Need to know how design should be, and also what information it should have.

**Comment by samuveth** [comment]
I don't think we currently need this. I will make a new issue to improve the current Notifications design and language.
